doc/source/whatsnew/v0.20.0.txt

```diff
@@ -398,6 +399,39 @@ To convert a ``SparseDataFrame`` back to sparse SciPy matrix in COO format, you
 
    sdf.to_coo()
 
+.. _whatsnew_0200.enhancements.style_excel:
+
+Excel output for styled DataFrames
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Experimental support has been added to export ``DataFrame.style`` formats to Excel using the ``openpyxl`` engine. (:issue:`15530`)
+
+For example, after running the following, ``styled.xlsx`` renders as below:
+
+.. ipython:: python
+
+    np.random.seed(24)
+    df = pd.DataFrame({'A': np.linspace(1, 10, 10)})
+    df = pd.concat([df, pd.DataFrame(np.random.RandomState(24).randn(10, 4),
+                                     columns=list('BCDE'))],
+                   axis=1)
+    df.iloc[0, 2] = np.nan
+    df
+    styled = df.style.\
+        applymap(lambda val: 'color: %s' % 'red' if val < 0 else 'black').\
+        apply(lambda s: ['background-color: yellow' if v else ''
+                         for v in s == s.max()])
+    styled.to_excel('styled.xlsx', engine='openpyxl')
+
+.. image:: _static/style-excel.png
+
+.. ipython:: python
+    :suppress:
+    import os
+    os.remove('styled.xlsx')
+
+See the :ref:`Style documentation <style>` for more detail.
```

Comment:
This ref isn't working. @TomAugspurger, I assume this is related to the nbsphinx changes. There is another ref to `style` in 0.17.1